pkg/service/service.go

```diff
@@ -225,6 +227,7 @@ func (s *deviceService) LoadCustomConfig(customConfig interfaces.UpdatableConfig
 	}
 
 	s.controller.SetCustomConfigInfo(customConfig)
```

Comment:
Can this now be removed since it was for the common Common endpoint?

Reply:
custom config is different from common config
it's for the special config in different device services

Reply:
Right... ;-) THX!